src/iOS/iOS.csproj

```diff
@@ -153,6 +152,19 @@
     <ImageAsset Include="Resources\Assets.xcassets\LaunchScreen.imageset\logo_white%403x.png" />
     <ImageAsset Include="Resources\Assets.xcassets\ic_warning.imageset\Contents.json" />
     <ImageAsset Include="Resources\Assets.xcassets\ic_warning.imageset\ic_warning-1.pdf" />
+    <BundleResource Include="Resources\bwi-font.ttf" />
```

Comment:
Does this change in organization matter? (vs. the previous individual `ItemGroup`)

@vvolkgang @jlf0dev @fedemkr

Reply:
I'm pretty sure we're good. I also found this in the [documentation](https://docs.microsoft.com/en-us/visualstudio/msbuild/itemgroup-element-msbuild?view=vs-2022):

> In a simple project file, you normally use a single ItemGroup element, but you can also use multiple ItemGroup elements. When multiple ItemGroup elements are used, items are combined into a single ItemGroup. For example, some items might be included by a separate ItemGroup element that's defined in an imported file.

Reply:
MSBuild wise the end result will be the same. 

@Hinton which IDE did you use? I would suggest opening the .csproj in VS4Mac and Rider just to if this changes between those two. VS4Mac formats .csproj's differently than VS4win for instance, which leads to always having that pending change. We should follow the format used by our most common IDE.

Reply:
Visual Studio.